Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -56,8 +56,7 @@ class AppSettingsTableViewController: SettingsTableViewController {
         ]
 
         let accountChinaSyncSetting: [Setting]
-        let locale = NSLocale.currentLocale()
-        if locale.localeIdentifier != "zh_CN" {
+        if !(profile as! BrowserProfile).isChinaEdition {
```

Comment:
I have moved isChinaEdition into the Profile protocol, and provided a dummy implementation in MockProfile.
What do you think of this change?